nixos/modules/services/hardware/g810-led.nix

```diff
@@ -0,0 +1,49 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.g810-led;
+in
+
```

Comment:
extra line here? or is it something nixfmt-rfc-style did?

Reply:
Removed it.